libdnf/dnf-transaction.cpp

```diff
@@ -1387,7 +1387,6 @@ dnf_transaction_commit(DnfTransaction *transaction, HyGoal goal, DnfState *state
 
     /* no signature checking, we've handled that already */
     vs_flags = rpmtsSetVSFlags(priv->ts, _RPMVSF_NOSIGNATURES | _RPMVSF_NODIGESTS);
-    rpmtsSetVSFlags(priv->ts, vs_flags);
```

Comment:
Since you've removed this, wouldn't it make sense to eliminate the `vs_flags` variable, since it's not used anymore...

Reply:
Done